src/app/core/providers/effects/tags-effects.service.ts

```diff
@@ -14,31 +15,45 @@ export class TagsEffectsService {
                 private notification: I18nNotification,
                 private state: ApplicationStateService) {
     }
+    // Load tag families and their sibling tags for a project
+    loadTagFamiliesAndTheirTags(project: string): void {
+        this.state.actions.tag.clearAll();
+        this.state.actions.tag.actionStart();
+        this.api.project.getTagFamilies({ project })
+        .subscribe(tagFamiesResponse => {
+            this.state.actions.tag.fetchTagFamiliesSuccess(tagFamiesResponse.data);
+            tagFamiesResponse.data.forEach((tagFamily: TagFamily) => this.loadTagsOfTagFamily(project, tagFamily.uuid));
+        }, error => {
+            this.state.actions.tag.actionError();
+            this.notification.show({
+                type: 'error',
+                message: 'editor.load_tag_families_error'
+            });
+        });
+    }
 
-    createTagFamily(project: string, name: string): Promise<TagFamilyResponse > {
+    loadTagsOfTagFamily(project: string, tagFamilyUuid: string): void {
         this.state.actions.tag.actionStart();
```

Comment:
Don't use `actionStart()` - make a distinct action for each thing. Same with `actionError()`